test/IntegrationTests/MSTest.Acceptance.IntegrationTests/RunsettingsTests.cs

```diff
@@ -48,6 +71,55 @@ public async Task UnsupportedRunSettingsEntriesAreFlagged(string tfm)
+                testHostResult.AssertOutputContains("Les attributs Runsettings « TargetFrameworkVersion » ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
+                testHostResult.AssertOutputContains("Les attributs Runsettings « TargetPlatform » ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
+                testHostResult.AssertOutputContains("Les attributs Runsettings « TestAdaptersPaths » ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
+                testHostResult.AssertOutputContains("Les attributs Runsettings « TestCaseFilter » ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
+                testHostResult.AssertOutputContains("Les attributs Runsettings « TestSessionTimeout » ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
+                testHostResult.AssertOutputContains("Les attributs Runsettings « TreatNoTestsAsError » ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
+                break;
+            case "it-IT":
+                testHostResult.AssertOutputContains("I logger Runsettings non sono supportati da Microsoft.Testing.Platform e verranno ignorati");
+                testHostResult.AssertOutputContains("I datacollector Runsettings non sono supportati da Microsoft.Testing.Platform e verranno ignorati");
+
+                // Unsure why this happens :/
+                string notSupportedItalianMessageFormat = OperatingSystem.IsWindows()
+                    ? "L'attributo Runsettings `{0}' non è supportato da Microsoft.Testing.Platform e verrà ignorato"
+                    : "L’attributo Runsettings ‘0’ non è supportato da Microsoft.Testing.Platform e verrà ignorato";
+
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "MaxCpuCount"));
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "TargetFrameworkVersion"));
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "TargetPlatform"));
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "TestAdaptersPaths"));
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "TestCaseFilter"));
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "TestSessionTimeout"));
+                testHostResult.AssertOutputContains(string.Format(CultureInfo.InvariantCulture, notSupportedItalianMessageFormat, "TreatNoTestsAsError"));
```

Comment:
I am perfectly fine with asserting only one line, hopefully that will reduce the testing pain diff for some symbols:

```suggestion
            case "fr-FR":
                testHostResult.AssertOutputContains("Les loggers Runsettings ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
                testHostResult.AssertOutputContains("Les datacollecteurs Runsettings ne sont pas pris en charge par Microsoft.Testing.Platform et seront ignorés");
                break;
            case "it-IT":
                testHostResult.AssertOutputContains("I logger Runsettings non sono supportati da Microsoft.Testing.Platform e verranno ignorati");
                testHostResult.AssertOutputContains("I datacollector Runsettings non sono supportati da Microsoft.Testing.Platform e verranno ignorati");
```

Reply:
I was going to switch `it-IT` with `cs-CZ`, but asserting one/two lines look good as well and perfectly does the purpose we need.